<?xml version="1.0"?>

<launch>
		<include file="$(find airship_keyboard)/launch/harcom_runalone.launch"/>
		<include file="$(find airship_keyboard)/launch/camcom_runalone.launch"/>
		<include file="$(find svo_ros)/launch/live.launch"/>

		<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="odometry_runalone">
    		<param name="output_frame" value="odom"/>
    		<param name="freq" value="30.0"/>
    		<param name="sensor_timeout" value="1.0"/>
    		<param name="odom_used" value="false"/>
    		<param name="imu_used" value="true"/>
    		<param name="vo_used" value="true"/>
    		<param name="debug" value="false"/>
    		<param name="self_diagnose" value="false"/>

    		<remap from="imu" to="hardcom/InertialData" />
    		<remap from="vo" to="svo/InertialData" />

  		</node>

        <node pkg="airship_keyboard" type="pose2odometry.py" name="pose2odometry"  output="screen">
        		<param name="rate" value="30.0" type="double"/>
				<param name="topic" value="odometry" type="string"/>
				<param name="topic_hardcom" value="hardcom" type="string"/>
				<param name="topic_camcom" value="cameracom" type="string"/>
				<param name="topic_command" value="command" type="string"/>
        </node>
</launch>
